haskell-indentation.el

```diff
@@ -87,18 +100,13 @@
 It rebinds RET, DEL and BACKSPACE, so that indentations can be
 set and deleted as if they were real tabs."
   :keymap haskell-indentation-mode-map
-  (kill-local-variable 'indent-line-function)
```

Comment:
Right, I was thinking that someone would always want either haskell-indent or haskell-indentation. I guess someone might want neither.